src/components/Media/Media.tsx

```diff
@@ -35,40 +29,27 @@ const Media: React.FC<MediaProps> = ({ items, activeItem }) => {
   return (
     <MediaWrapper>
       {items.map((item: object, key: number) => {
-        // 1. get canvas
-        const canvas: CanvasNormalized = vault.fromRef(item);
-        // 2. get annotationPage
-        const annotationPage: AnnotationPageNormalized = vault.fromRef(
-          canvas.items[0],
-        );
-        // 3.get annotations
-        const annotations: Array<AnnotationNormalized> = vault.allFromRef(
-          annotationPage.items,
+        // this probably needs to be written in a .filter()
+        const canvasEntity: object = getCanvasByCriteria(
```

Comment:
Can we type `canvasEntity` with it's interface from the service helper, instead of using `object`?

Reply:
Good call. I just removed the object type and it now maintains the CanvasEntity type returned from the helper. 
![image](https://user-images.githubusercontent.com/7376450/135510932-2a6cbe54-abd5-441f-8715-f115387fc0ed.png)